seed/static/seed/partials/program_setup.html

```diff
@@ -44,7 +44,7 @@ <h2><i class="fa fa-tachometer"></i> {$:: 'Program Metric Configuration' | trans
                             </li>
                         </ul>
                         <div class="r-panel r-info r-margin-top-10">
-                            Configure your program metric to enable visualizations on the <b><a ui-sref="insights_program(::{organization_id: org.id})" ui-sref-active="active" translate>program insights</a></b> page.
+                            Configure your program metric to enable visualizations on the <b><a ui-sref="insights_program(::{organization_id: org.id})" ui-sref-active="active" translate>program overview</a></b> page.
```

Comment:
These changes were due to my mistake of accidentally pushing code direct to the 'bps-insights-development' branch instead of a separate branch. I went ahead and removed that commit from the 'bps' branch and add it to my own branch and a PR so we should probably remove these changes in your PR, right?